tox.ini

```diff
@@ -0,0 +1,9 @@
+[tox]
+envlist = foxpuppet
```

Comment:
I would make the envlist `py{27,35}` so that we run the tests against python 2.7 and 3.5. Later I would also introduce a `flake8` environment, but let's keep it to the tests for now.